src/main/webapp/js/instructorCourseEnrollPage.js

```diff
@@ -10,7 +10,7 @@ var loadUpFunction = function () {
 
     var ENTER_KEYCODE = 13;
     var enrolTextbox; 
-    if ((enrolTextbox	 = $('#enrollstudents')).length){
+    if ((enrolTextbox     = $('#enrollstudents')).length){
```

Comment:
Yes, that's exactly what I had in mind. If I add all the rules at once the diff will easily reach a 4-5 digit number.